demos/code/code.js

```diff
@@ -383,10 +383,20 @@ Code.init = function() {
   };
   window.addEventListener('resize', onresize, false);
 
-  // Interpolate translated messages into toolbox.
+  // The toolbox XML specifies each category name using Blockly's messaging
+  // format (eg. `<category name="%{BKY_CATLOGIC}">`).
+  // These message keys need to be defined in `Blockly.Msg` in order to
+  // be decoded by the library. Therefore, we'll use the `MSG` dictionary that's
+  // been defined for each language to import each category name message 
+  // into `Blockly.Msg`.
+  for (var messageKey in MSG) {
```

Comment:
Out of scope for this CL, but please add a TODO to clean up how this demo adds %{} messages to do it explicitly instead of searching for messages that start with cat.